server.go

```diff
@@ -1556,7 +1556,7 @@ func newServer(cfg *Config, listenAddrs []net.Addr,
 			}
 
 			br, err := lnwallet.NewBreachRetribution(
-				channel, commitHeight, 0, nil,
+				channel, commitHeight, 0, nil, dbs.AuxLeafStore,
```

Comment:
We also need to hook up the actual aux leaf store in a couple of other places:
 - In all calls to `lnwallet.NewLightningChannel()` (via `WithLeafStore()`)
 - In `chainWatcherConfig.auxLeafStore` in `ChainArbitrator.Start()`
 - In `chainWatcherConfig.auxLeafStore` in `ChainArbitrator.WatchNewChannel()`